### DISCUSSION TITLE
Wasteland

### DISCUSSION THREAD

**Discussion by Kor-Angar** [dosinabox/g2nr_unofficial_update#359] [General]
Что бы расширенный мир в миненталя отключить достаточно файл удалить?

**Comment by dosinabox** [comment]
Да, но только до первого посещения. После первой загрузки весь мир будет записан в сохраненку при сохранении (даже если вернуться в Хоринис и сохраниться там).

**Comment by Kor-Angar** [comment]
ясно. надо будет подумать что делать с новой территорией. так как там за монстров дают опыт и это не очень хорошо

**Comment by dosinabox** [comment]
При установленном расширенном мире можно давать меньше опыта за всех монстров, пока игрок находится там. Например.

**Comment by Kor-Angar** [comment]
или просто давать по 0 опыта за всех орков что там ;-)

**Comment by Kor-Angar** [comment]
кровати в замке у магов нет
